clap_complete/tests/snapshots/two_multi_valued_arguments.zsh

```diff
@@ -0,0 +1,30 @@
+#compdef my-app
+
+autoload -U is-at-least
+
+_my-app() {
+    typeset -A opt_args
+    typeset -a _arguments_options
+    local ret=1
+
+    if is-at-least 5.2; then
+        _arguments_options=(-s -S -C)
+    else
+        _arguments_options=(-s -C)
+    fi
+
+    local context curcontext="$curcontext" state line
+    _arguments "${_arguments_options[@]}" \
+'-h[Print help]' \
+'--help[Print help]' \
+'*::first -- first multi-valued argument:' \
```

Comment:
This is the key change in behavior. `second` is intentionally omitted.

Reply:
To help highlight this, would be you willing to split this into two commits
- The first commit adds the tests and they pass, showing the existing behavior
- The second commit adds the fix and updates the test snapshots, showing the new behavior